ui/apps/platform/src/Containers/Clusters/ClusterPage.tsx

```diff
@@ -312,18 +299,12 @@ function ClustersSidePanel({ selectedClusterId }: ClustersSidePanelProps): React
         );
 
     return (
-        <SidePanelAnimatedArea isDarkMode={isDarkMode} isOpen={!!selectedClusterId}>
+        <section className="flex flex-1 flex-col h-full">
+            <PageHeader header={selectedClusterName} subHeader="Cluster">
+                <PanelHeadEnd>{panelButtons}</PanelHeadEnd>
+            </PageHeader>
+
             <PanelNew testid="clusters-side-panel">
```

Comment:
It may not be worth the change due to possible side effects and/or additional effort, but similar comment here regarding the naming of `testid`.